tests/packagedcode/data/alpine/alpine-container-layer.tar.xz-get-installed-expected.json

```diff
@@ -32,7 +33,7 @@
     "code_view_url": null,
     "vcs_url": "git+http://git.alpinelinux.org/aports/commit/?id=cb70ca5c6d6db0399d2dd09189c5d57827bce5cd",
     "copyright": null,
-    "license_expression": "gpl-2.0",
+    "license_expression": "unknown",
```

Comment:
The expected license expression should not be changed.

You may need to run `scancode --reindex-licenses` before regenerating the test results.